src/main/java/graphql/validation/rules/UniqueVariableNamesRule.java

```diff
@@ -0,0 +1,47 @@
+package graphql.validation.rules;
+
+import graphql.language.*;
```

Comment:
please have a look at the intellij formatter inside the root folder: we don't use `*` imports.

Reply:
Thanks for you review and guide, I have updated intellij setting and PR.